[2022-11-04T18:59:00.250+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: orquestacion.TASK1 scheduled__2022-05-11T00:00:00+00:00 [queued]>
[2022-11-04T18:59:00.282+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: orquestacion.TASK1 scheduled__2022-05-11T00:00:00+00:00 [queued]>
[2022-11-04T18:59:00.284+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-04T18:59:00.286+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-04T18:59:00.287+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-04T18:59:00.335+0000] {taskinstance.py:1383} INFO - Executing <Task(BashOperator): TASK1> on 2022-05-11 00:00:00+00:00
[2022-11-04T18:59:00.349+0000] {standard_task_runner.py:55} INFO - Started process 4700 to run task
[2022-11-04T18:59:00.355+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'orquestacion', 'TASK1', 'scheduled__2022-05-11T00:00:00+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/5.1-orquestacion.py', '--cfg-path', '/tmp/tmpfdqetbnc']
[2022-11-04T18:59:00.356+0000] {standard_task_runner.py:83} INFO - Job 33: Subtask TASK1
[2022-11-04T18:59:00.498+0000] {task_command.py:376} INFO - Running <TaskInstance: orquestacion.TASK1 scheduled__2022-05-11T00:00:00+00:00 [running]> on host c7d12af55512
[2022-11-04T18:59:00.654+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=orquestacion
AIRFLOW_CTX_TASK_ID=TASK1
AIRFLOW_CTX_EXECUTION_DATE=2022-05-11T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-11T00:00:00+00:00
[2022-11-04T18:59:00.657+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2022-11-04T18:59:00.660+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', "SLEEP 2 && ECHO 'TASK1'"]
[2022-11-04T18:59:00.685+0000] {subprocess.py:86} INFO - Output:
[2022-11-04T18:59:00.688+0000] {subprocess.py:93} INFO - /bin/bash: line 1: SLEEP: command not found
[2022-11-04T18:59:00.689+0000] {subprocess.py:97} INFO - Command exited with return code 127
[2022-11-04T18:59:00.713+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f'Bash command failed. The command returned a non-zero exit code {result.exit_code}.'
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 127.
[2022-11-04T18:59:00.721+0000] {taskinstance.py:1406} INFO - Marking task as FAILED. dag_id=orquestacion, task_id=TASK1, execution_date=20220511T000000, start_date=20221104T185900, end_date=20221104T185900
[2022-11-04T18:59:00.751+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 33 for task TASK1 (Bash command failed. The command returned a non-zero exit code 127.; 4700)
[2022-11-04T18:59:00.776+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2022-11-04T18:59:00.836+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
